straxen/contexts.py

```diff
@@ -218,19 +219,98 @@ def xenonnt_led(**kwargs):
     return st
 
 
-def xenonnt_simulation(output_folder='./strax_data'):
+def xenonnt_simulation(output_folder='./strax_data',
+                       cmt_run_id='020280',
+                       cmt_version='v3',
+                       fax_config='fax_config_nt_design.json',
+                       cmt_option_overwrite=immutabledict(),
```

Comment:
So, if we provide electron lifetime here, will it overwrite it for wfsim, but not during straxen processing, that will use value from `cmt_run_id`?

Reply:
It's the other way, that the `cmt_option_overwrite` can only overwrite CMT options in processing or `gain_model_mc`.
If you want to use a user-specific electron lifetime in simulation, you should delete `electron_lifetime_liquid='elife_conf'` from `_config_overlap` and set the lifetime in `fax_config_override`